packages/web/docs/src/content/gateway/deployment/node-frameworks/nestjs.mdx

```diff
@@ -0,0 +1,62 @@
+---
+description:
+  Nest (Nest JS) is a progressive Node.js framework for building efficient, reliable and scalable
+  server-side applications.
```

Comment:
This is a description of Nest, not our Hive GW driver _for_ Nest. I'd rephrase it to say something like: "Hive Gateway driver for Nest, a progressive Node.js [...]"